lib/decorateWithMetrics.js

```diff
@@ -115,30 +127,55 @@ function toHumanReadableDuration(millis) {
  * Print extensive information on the dev console
  * max, min, average, total time for each method
  * and a table of individual calls
+ *
+ * @param {boolean} [raw=false] setting this to true will print raw instead of human friendly times
+ * Useful when you copy the printed table to excel and let excel do the number formatting
  */
-function printMetrics() {
-    const {totalTime, averageTime, summaries} = getMetrics();
+function printMetrics(raw = false) {
+    const {totalTime, summaries, lastCompleteCall = {endTime: -1}} = getMetrics();
```

Comment:
This is a precaution for when `printMetrics` is called before any calls are made at all
Since `lastCompleteCall` is derived from `calls` data, and there are none it will be `undefined` 
I've picked `-1` as it looks clear that no call has happened, I guess `0` can work as well

Reply:
Ah, I see. In that case, I could expect to see something like `"Unavailable"`, but the solution you have here works.